### PR TITLE
File upload fix

### PR DESCRIPTION
**Pull request by Boyko-Karadzhov**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,32 +20,41 @@
   "license": "MIT",
   "homepage": "https://github.com/payhawk/xero-integration#readme",
   "devDependencies": {
-    "@types/jest": "24.9.0",
+    "@types/jest": "25.1.4",
     "@types/mime-types": "2.1.0",
-    "@types/node": "12.12.24",
+    "@types/node": "^12.12.29",
```
Comment (igrantbg) [inline package.json]: maybe pin all the versions

Comment (vassildinev) [inline package.json]: maybe pin all the versions

File: src/utils/document-sanitizer/DocumentSanitizer.ts [hunk 2]
```diff
@@ -0,0 +1,58 @@
+import fs = require('fs');
+import { exec } from 'child_process';
+
+import Jimp = require('jimp');
+import { IDocumentSanitizer } from './IDocumentSanitizer';
+
+// const FILE_SIZE_LIMIT = 3 * 1024 * 1024;
```
Comment (vassildinev) [inline src/utils/document-sanitizer/DocumentSanitizer.ts]: This is the real file size limit, right

File: package.json (hunk 1 above)
Comment (Boyko-Karadzhov) [reply package.json]: Not pinning those that we want to keep the major version of.

File: src/utils/document-sanitizer/DocumentSanitizer.ts [hunk 3]
```diff
@@ -0,0 +1,57 @@
+import fs = require('fs');
+import { exec } from 'child_process';
+
+import Jimp = require('jimp');
+
+import { IDocumentSanitizer } from './IDocumentSanitizer';
+
+const FILE_SIZE_LIMIT = 3 * 1024 * 1024;
+
+export class DocumentSanitizer implements IDocumentSanitizer {
```
Comment (vassildinev) [inline src/utils/document-sanitizer/DocumentSanitizer.ts]: A couple of unit tests?

File: src/utils/document-sanitizer/DocumentSanitizer.ts [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+    async sanitize(input: string) {
+        const fileSize = this.getFileSize(input);
+        if (fileSize < FILE_SIZE_LIMIT) {
```
Comment (vassildinev) [inline src/utils/document-sanitizer/DocumentSanitizer.ts]: if file size is exaxtly FILE_SIZE_LIMIT, the ratio will be 1 and image resize will do work without shrinking

File: src/utils/document-sanitizer/DocumentSanitizer.ts (hunk 3 above)
Comment (Boyko-Karadzhov) [reply src/utils/document-sanitizer/DocumentSanitizer.ts]: It's all magic. I have no expectation of my code

Comment (vassildinev) [reply src/utils/document-sanitizer/DocumentSanitizer.ts]: :D I mean covering the file extensions logic

**Pull request merged**